### ISSUE TITLE
[BUG] java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3

### ISSUE THREAD

**Issue by MADrewry** [snowgears/shopbugs#426]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Type of bug

Error in console

### Environment

```markdown
- Server Version: Paper 1.20.6-137-master@bd5867a
- Shop Version: 1.8.4.7
```

### Server startup log

https://paste.gg/p/anonymous/6e7319dc2e6448c8809a3447957b5ab3

### Shop config files

https://paste.gg/p/anonymous/96a31fc46742468287cbcd75dc2f635e

### Bug description

When Shop attempts to enable it causes error: [Server thread/ERROR]: Error occurred while enabling Shop v1.8.4.7 (Is it up to date?)
java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3

### Steps to reproduce

Put Shop 1.8.4.7 plugin in plugins directory
Start minecaft server

### Expected behaviour

Shop plugin should load and enable successfully

### Actual behaviour

Shop causes error and does not enable

### Other information

- EDIT - OOPS sorry, didn't search enough, this is actually a duplicate of issue #425

**Comment by HammontreeArt** [comment]
I'm getting the same bug. 

[00:59:58 ERROR]: Error occurred while enabling Shop v1.8.4.7 (Is it up to date?)
java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
        at Shop-1.8.4.7.jar/com.snowgears.shop.util.NMSBullshitHandler.init(NMSBullshitHandler.java:28) ~[Shop-1.8.4.7.jar:?]
        at Shop-1.8.4.7.jar/com.snowgears.shop.util.NMSBullshitHandler.<init>(NMSBullshitHandler.java:21) ~[Shop-1.8.4.7.jar:?]
        at Shop-1.8.4.7.jar/com.snowgears.shop.Shop.onEnable(Shop.java:184) ~[Shop-1.8.4.7.jar:?]



[config.txt](https://github.com/user-attachments/files/15814016/config.txt)
[20240612_ShopPlugin_IndexOutofRange.txt](https://github.com/user-attachments/files/15814017/20240612_ShopPlugin_IndexOutofRange.txt)

**Comment by OstlerDev** [comment]
Same issue with Paper 1.21 Experimental

**Comment by OstlerDev** [comment]
Looks like the issue is with Paper itself, the author posted this update over on the Spigot plugin page.

> Paper introduced a major limitation in 1.20.5/6. Plugins can no longer load files at runtime or check the Minecraft version which is how shop loads the correct file versions.

https://www.spigotmc.org/threads/shop-the-intuitive-shop-plugin-paid.79064/page-168#post-4739435

**Comment by OstlerDev** [comment]
This issue has been fixed in Shop 1.9.0.0
